### ISSUE TITLE
Question: Flashable image for STM32?

### ISSUE THREAD

**Issue by retsifp** [davidgfnet/wifi_display#4]
Hello!
I really love your project, in fact I already bought one of these displays and as soon as I have some time, I will build one myself.

But I have one question: Is there a chance, that you provide a compiled image for the STM32? :smile: 
As far as I can see, I only need to modify the ESP8266 code for my needs, so this would save me a lot of time :wink:

**Comment by davidgfnet** [comment]
Well yeah, but it should be straightforward for you to build the firmware yourself.
You just need a baremetal gcc arm toolchain and do a "make" command. No weird dependencies.

**Comment by retsifp** [comment]
Just FTR, I only had to install `gcc-arm-none-eabi` on my Debian system and run a `make` inside the `stm32_application` folder. 

There were some errors, I could fix them by adding `-std=c99` to the `CFLAGS` inside Makefile and adding `#define asm __asm` to main.c and gde043a2.c (I know, it's a dirty hack, but it works :wink:).

Now I have a `firmware.bin`, I hope flashing works... :wink:

**Comment by davidgfnet** [comment]
Yes you are absolutely right, c99 is missing in the Makefile, the reason it works for me is cause I use gcc5 probably (and c11 seems to be the default mode). About the __asm bit I have no idea, I guess has something to do with the compiler version too :) Thanks!
